.github/workflows/test.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         run: pip install pipenv==2023.7.23
       - name: Create virtualenv
         run: pipenv sync
+      - name: Install all missing stub packages
+        run: pipenv run mypy --install-types
```

Comment:
i think you can just install it with `pipenv install` so it's added to the pipfile and lock.

Reply:
How so? `requests` for instance is already in the pipfile, but the library stubs still seem to be missing.